host-controller/src/main/java/org/jboss/as/domain/controller/resources/ServerGroupResourceDefinition.java

```diff
@@ -73,6 +73,12 @@
             .addAccessConstraint(SensitiveTargetAccessConstraintDefinition.SOCKET_BINDING_REF)
             .build();
 
+    public static final SimpleAttributeDefinition SOCKET_BINDING_DEFAULT_INTERFACE = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.SOCKET_BINDING_DEFAULT_INTERFACE, ModelType.STRING, true)
+            .setAllowExpression(true)
```

Comment:
This is a model reference attribute, and we don't allow expressions in those (except for a few early mistakes that we haven't reverted due to compatibility concerns.)